### PR TITLE
Biotransformer

### PR DESCRIPTION
**Pull request by smartx-usman**
This PR is to add biotransformer wrapper and deploy it to Toolshed.

### CODE REVIEW

File: tools/biotransformer/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+categories:
+- Metabolomics
+description: BioTransformer is a tool for prediction of small molecule metabolism in mammals.
+homepage_url: https://gitlab.ics.muni.cz/recetox/mass-spectrometry/biotransformer
```
Comment (martenson) [inline tools/biotransformer/.shed.yml:4]: I believe this should point to the official webpage of the software - i.e. we should probably use https://bitbucket.org/djoumbou/biotransformerjar from https://jcheminf.biomedcentral.com/articles/10.1186/s13321-018-0324-5

File: tools/biotransformer/.shed.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+long_description: |
+  This tool is a complete wrapper for biotransformer.
```
Comment (martenson) [inline tools/biotransformer/.shed.yml:6]: It is useful to provide a paragraph of what the tool does on what type of data to help future users. We could take it from the tool's `<help>` section.

File: tools/biotransformer/biotransformer.xml [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+(InChI, InChIKey, Synonyms, Molecular formula, Major Isotope Mass, AlogP) and third with filtered duplicates based on 3 columns (Molecular fo
+rmula, Major Isotope Mass, AlogP).
+
+Output of BioTransformer with SMILES as an input is 1 CSV file.
+
+(* ) While the 'superbio' option runs a set number of transformation steps in a pre-defined order (e.g. deconjugation first, then Oxidation/r
+eduction, etc.), the 'allHuman' option predicts all possible metabolites from any applicable reaction(Oxidation, reduction, (de-)conjugation)
+ at each step.
+
+Cite: Djoumbou-Feunang Y, Fiamoncini J, de la Fuente AG, Manach C, Greiner R, and Wishart DS; BioTransformer: A Comprehensive Computational T
```
Comment (martenson) [inline tools/biotransformer/biotransformer.xml:127]: We could use the `<citation>` galaxy tool element here to render the citation in bibtex. See https://docs.galaxyproject.org/en/master/dev/schema.html#tool-citations

**Review by martenson — APPROVED** [review]
It looks alright to me, thanks! One thing we are missing is a tool test. Given the current secretive location of the docker image our github CI has no means to execute it at the moment - however we should make effort for our tools to be testable as soon as we can.

File: tools/biotransformer/biotransformer.xml (hunk 3 above)
Comment (smartx-usman) [reply tools/biotransformer/biotransformer.xml]: Added citation tag as suggested.

File: tools/biotransformer/.shed.yml (hunk 2 above)
Comment (smartx-usman) [reply tools/biotransformer/.shed.yml]: Long description is updated to give some meaningful info to the users.

File: tools/biotransformer/.shed.yml (hunk 1 above)
Comment (smartx-usman) [reply tools/biotransformer/.shed.yml]: It's changed to the official webpage of the software in the recent commit.

**Comment by martenson** [comment]
Thanks a lot, looks good!

I merged this manually, see https://github.com/RECETOX/galaxytools/commits/master

**Pull request closed without merging**